### ISSUE TITLE
ReduceIndex In GraphQL, how to implement queries？

### ISSUE THREAD

**Issue by chinasqzl** [OrchardCMS/OrchardCore#11740]
public class IndexPropertyProvider<T> : IIndexPropertyProvider where T : MapIndex

ReduceIndex In GraphQL, how to implement queries？

**Comment by sebastienros** [comment]
Can you rephrase? Maybe provide an example.
If it is about using graphql on a Table created my a migration and a reduce index (yessql) then each of those need to be registered specifically.

We could imagine a feature that would go over each index (map or reduce) and maps them as a graphql type.

**Comment by chinasqzl** [comment]
Example scenario ： Query users by role

The role is ReduceIndex
